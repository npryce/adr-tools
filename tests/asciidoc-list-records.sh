echo "adoc" > .adr-file-type
adr list
adr new first
adr list
adr new second
adr list
adr new third
adr list

