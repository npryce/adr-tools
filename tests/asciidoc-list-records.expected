echo "adoc" > .adr-file-type
adr list
The doc/adr directory does not exist
adr new first
doc/adr/0001-first.adoc
adr list
doc/adr/0001-first.adoc
adr new second
doc/adr/0002-second.adoc
adr list
doc/adr/0001-first.adoc
doc/adr/0002-second.adoc
adr new third
doc/adr/0003-third.adoc
adr list
doc/adr/0001-first.adoc
doc/adr/0002-second.adoc
doc/adr/0003-third.adoc

